02-getting-account-data/index.js

```diff
@@ -0,0 +1,38 @@
+import { SDK } from 'NEWSDKNAME'
+
+// SDK exposes all you need like anchor-link does, users don't need to import from core directly, if they need something they do
+// import { Account } from 'NEWSDKNAME'
+
+const sdk = new SDK({ url: 'https://eos.greymass.com' }) // loaded with default plugins (something like if (browser) { plugins = [anchor-browser, atomichub] })
+
+async function main() {
+    const session = await sdk.login('myapp') // login also restores existing session unlike anchor-link, can have option for forcing re-login
+
+    // Loads account
+    const account = await session.accounts[0].get()
+    // Or you can load the account from the account name with either
+    // const account = await session.accounts.get('teamgreymass')
+    // or
+    // const account = await Account.get('eos', 'teamgreymass')
+
+    // Loads account balance
+    const balance = await account.getBalance('EOS')
```

Comment:
How would it know the token contract? 🤔 

If I wanted to do:

```js
account.getBalance('USDT')
```

I'm not sure how it'd know. Maybe instead, if you call: 
```js
account.getBalance()
``` 

it would load the default token from the system contract, but otherwise you have to pass the contract name and token name, e.g. 

```js 
account.getBalance('eosio.token', 'EOS')
```

Reply:
Yeah that makes sense!

Reply:
I don't think this makes sense to have here since the token contracts are not even standardized

Reply:
Yeah that's true. Why not just have a `account.getBalance('USDT')` method that would just assume that the token is on the system contract?

Reply:
What if instead of dealing with balances on account objects, we dealt with them on contract objects?

Each token is a part of a contract. It changes the paradigm of how you load it, but maybe we can still make sure it's just as easy?

Reply:
Removed this from the example here.